ledger/sandbox/src/main/resources/db/migration/postgres/V28__Pruned_offset.sql

```diff
@@ -0,0 +1,12 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V28: Pruned offset tracking
+--
+-- This schema version adds tracking of the pruned portion of the participant ledger to the
+-- parameters table.
+---------------------------------------------------------------------------------------------------
+
+-- Add the column for most recent pruning offset to parameters.
+ALTER TABLE parameters ADD COLUMN participant_pruned_up_to_inclusive bytea;
```

Comment:
```suggestion
ALTER TABLE parameters ADD COLUMN participant_pruned_up_to_inclusive BYTEA;
```

Reply:
Also appending `using decode(lpad(to_hex(participant_pruned_up_to_inclusive), 16, '0'), 'hex')` again to be consistent with `parameters.ledger_end`

Reply:
Changed to BINARY to be consistent with `parameters.ledger_end`.